Voycar.Api.Gps/Program.cs

```diff
@@ -0,0 +1,6 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
```

Comment:
Warum ist der "Hello World" Endpoint hier in `Voycar.Api.Gps`? Haben wir jetzt unser Beispiel im GPS-Projekt angelegt?

Reply:
Weiß ich auch nicht

Reply:
Ist nur automatisch generiert, wird noch geändert ⇾ andere us.